suitcase/mongo_embedded/__init__.py

```diff
@@ -300,21 +323,23 @@ def datum(self, doc):
         return doc
 
     def event_page(self, doc):
-        self._bulkwrite_event({doc['descriptor']: doc})
+        doc_size = len(bson.BSON.encode(doc))
+        self._bulkwrite_event({doc['descriptor']: doc},
+                              {doc['descriptor']: doc_size})
         return doc
 
     def datum_page(self, doc):
-        self._bulkwrite_datum({doc['resource']: doc})
+        doc_size = len(bson.BSON.encode(doc))
+        self._bulkwrite_datum({doc['resource']: doc},
+                              {doc['resource']: doc_size})
         return doc
 
     def close(self):
         self.freeze(self._run_uid)
 
     def freeze(self, run_uid):
```

Comment:
Should we retire the name "freeze" now? As your changes to the docstring suggestion, "finalize" might be a good name for what this now does. I have seen that term used in similar contexts.

Reply:
sounds good finalize is better

Reply:
updated